bundle/bundle.go

```diff
@@ -1261,13 +1377,25 @@ func mktree(path []string, value interface{}) (map[string]interface{}, error) {
 // will have an empty revision except in the special case where a single bundle is provided
 // (and in that case the bundle is just returned unmodified.)
 func Merge(bundles []*Bundle) (*Bundle, error) {
+	return MergeWithRegoVersion(bundles, ast.RegoV0)
+}
+
+func MergeWithRegoVersion(bundles []*Bundle, regoVersion ast.RegoVersion) (*Bundle, error) {
```

Comment:
A description of how we decide the global rego version of the resulting bundle would be helpful. Can we merge a v0 and v1 bundle?

Reply:
The resulting bundle would get the provided `regoVersion` rego-version, and any Rego file with a different rego-version (either coming from the origin bundle's global rego-version, or from it's per-file manifest override), will get it's version registered in the manifest.